applications/StructuralMechanicsApplication/custom_response_functions/response_utilities/stress_response_definitions.cpp

```diff
@@ -20,77 +20,441 @@
 
 namespace Kratos
 {
-    namespace StressResponseDefinitions
+
+namespace StressResponseDefinitions
+{
+    TracedStressType ConvertStringToTracedStressType(const std::string& Str)
+    {
+        if(Str == "FX")
```

Comment:
why dont you save this in a map?
Seems to me like the more natural way

Reply:
Here i only fixed the indention level, so it was done like this already before. But I can of course do that in this PR.

Reply:
I thought about it, and i don't think it makes much sense to first fill a map with the value that I'm then looking for. (Also because of the overhead to create the map, but thats no major reason).

Reply:
you could fill the map with an initializer list

But it is up to you, its not that big of a difference

One adv is that you could loop the map and print the available options in case sth wrong is selected